Code/Framework/AzToolsFramework/AzToolsFramework/UI/Prefab/PrefabIntegrationManager.cpp

```diff
@@ -341,7 +341,8 @@ namespace AzToolsFramework
             }
 
             // Instantiate Prefab
-            if (selectedEntities.size() == 1 && !readOnlyEntityInSelection && !onlySelectedEntityIsClosedPrefabContainer)
+            if (selectedEntities.size() == 0 ||
+                selectedEntities.size() == 1 && !readOnlyEntityInSelection && !onlySelectedEntityIsClosedPrefabContainer)
```

Comment:
uber-nit: operator precedence should ensure this will resolve as intended, but for clarity I would add some parentheses:

```
if (selectedEntities.size() == 0 ||
    (selectedEntities.size() == 1 && !readOnlyEntityInSelection && !onlySelectedEntityIsClosedPrefabContainer))
```